gap/PolygonalComplexes/modification.gd

```diff
@@ -998,6 +998,42 @@ DeclareOperation("JoinVertexEdgePathsNC",
    DeclareOperation( "JoinBoundariesNC", [IsPolygonalComplex, IsList, IsPolygonalComplex, IsList, IsInt] );
 #! @EndGroup
 
+#! @BeginGroup JoinBoundary
+#! @Description
+#! This method takes a boundary vertex <A>v</A> and joins the vertex-edge-path of the 
+#! boundary where <A>v</A> lies on if the path has even length. 
+#! That means, the method split the vertex-edge-path of the boundary in two parts. 
```

Comment:
Better: This means, the method splits the vertex-edge-path of the boundary in two parts.